docs/developer-docs/integrations/https-outcalls/https-outcalls-how-to-use.md

```diff
@@ -36,51 +9,45 @@ It is expected that the majority of HTTP calls to be `GET` calls for reading Web
+* [Cycles](../../gas-cost.md) used to pay for the call must be explicitly transferred with the call. They are not deducted from the caller's balance implicitly.
 
 ## The API for sending HTTP outcalls
 
-As per the [Internet Computer interface specification](https://internetcomputer.org/docs/current/references/ic-interface-spec), a canister can use the `http_request` method by [following construction](https://internetcomputer.org/docs/current/references/ic-interface-spec#ic-http_request):
+As per the [Internet Computer interface specification](https://internetcomputer.org/docs/current/references/ic-interface-spec), a canister can use the `http_request` method by [following specification](https://internetcomputer.org/docs/current/references/ic-interface-spec#ic-http_request):
 
 ### The request
-The following parameters should be supplied for in the request:
+The following parameters should be supplied within the request:
 
--   `url`: the requested URL. The URL must be valid according to [RFC-3986](https://www.ietf.org/rfc/rfc3986.txt) and its length must not exceed `8192`. The URL may specify a custom port number.
+-   `url`: The requested URL that must be valid according to https://www.ietf.org/rfc/rfc3986.txt[RFC-3986] and its length must not exceed `8192`. The URL may specify a custom port number.
 
--   `max_response_bytes`: optional; specifies the maximal size of the response in bytes. If provided, the value must not exceed `2MB` (`2_000_000B`). The call will be charged based on this parameter. If not provided, the maximum of `2MB` will be used.
+-   `max_response_bytes`: Optional; Specifies the maximal size of the response in bytes. If provided, the value must not exceed `2MB` (`2_000_000B`). The call will be charged based on this parameter. If not provided, the maximum of `2MB` will be used.
 
--   `method`: currently, only `GET`, `HEAD`, and `POST` are supported.
+-   `method`: Currently, only `GET`, `HEAD`, and `POST` are supported.
 
--   `headers`: list of HTTP request headers and their corresponding values.
+-   `headers`: List of HTTP request headers and their corresponding values.
 
--   `body`: optional; the content of the request's body.
+-   `body`: Optional; The content of the request's body.
 
--   `transform`: an optional function that transforms raw responses to sanitized responses, and a byte-encoded context that is provided to the function upon invocation, along with the response to be sanitized. If provided, the calling canister itself must export this function.
+-   `transform`: An optional function that transforms raw responses to sanitized responses, and a byte-encoded context that is provided to the function upon invocation, along with the response to be sanitized. If provided, the calling canister itself must export this function.
```

Comment:
I suggest explaining what sanitized responses mean. Should we provide an example? Maybe giving an example of an API request that has a timestamp?

Reply:
Thanks! I added a link to information about sanitization and provided an example.